.github/CODEOWNERS

```diff
@@ -10,6 +10,7 @@
 /packages/@expo/metro-runtime                   @EvanBacon @bycedric @marklawlor
 /packages/@expo/package-manager                 @EvanBacon @bycedric
 /packages/@expo/plist                           @EvanBacon @bycedric
+/packages/@expo/pkcs12                          @bycedric
```

Comment:
Could you please add Quin as a codeowner, too?

Reply:
```suggestion
/packages/@expo/pkcs12                          @bycedric @quinlanj
```